Mathlib/Data/Multiset/DershowitzManna.lean

```diff
@@ -35,7 +38,7 @@ namespace Multiset
 
 variable {α : Type*}
 
-/-- The standard Dershowitz–Manna ordering: -/
+/-- The standard Dershowitz–Manna ordering. -/
 def DMLT [Preorder α] (M N : Multiset α) : Prop :=
```

Comment:
This abbreviation seems a bit obscure. What about
```suggestion
def IsDershowitzMannaLT [Preorder α] (M N : Multiset α) : Prop :=
```

Reply:
Sounds good! Thanks:-)